app/uk/gov/hmrc/eusubsidycompliancefrontend/views/AccountPage.scala.html

```diff
@@ -26,7 +26,7 @@
   govukInsetText : GovukInsetText
 )
 
-@(undertaking: Undertaking, isNonLeadEORIPresent: Boolean)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+@(undertaking: Undertaking, isNonLeadEORIPresent: Boolean, isTimeToReport: Option[Boolean], lastDayToReport: Option[String])(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
```

Comment:
Does `isTimeToReport` need to be an Option? Is defaulting a `None` to `false` sensible here instead?

Reply:
Both false and None means this message will not be displayed. I kept it Optional as it is derived from lastSubsidyUsageUpdt , which is optional and didn't want to do getOrElse :)
I can do getOrElse  to make it non optional

Reply:
updated